spec/mailers/employee_mailer_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe EmployeeMailer, type: :mailer do
+  describe 'delivery confirmation email sending' do
+    let(:order) { create(:order) }
+    let(:email) { described_class.send_delivery_confirmation_email(order) }
+
+    it 'renders the headers' do
+      expect(email.subject).to eq 'Reward has been delivered!'
+      expect(email.to).to eq [order.employee.email]
+      expect(email.from).to eq ['sz.nedzynski@gmail.com']
```

Comment:
I would consider adding it to some ENV variable, or app setting, since it appears in many places, and can ba hard to change later on

Reply:
Thanks! For consistency I put this e-mail data into rails credentials. Is it a good solution?